### DISCUSSION TITLE
yolo dataset format to paddleocr dataset format

### DISCUSSION THREAD

**Discussion by wdcs-krishpatel** [PaddlePaddle/PaddleOCR#13177] [Q&A] [answered]
I want to finetune paddleocr. Now I have image and label dataset in a yolov8 format, so how do I convert to padlleocr format and its folder structure and finetune it.

**Comment by GreatV** [comment]
please refer to: 
 https://github.com/PaddlePaddle/PaddleOCR/blob/433677182f108c1be413ee8a92815bc13b205737/tools/infer/predict_system.py#L124-L144

**Reply by wdcs-krishpatel** [reply]
but is this not for inference?

**Comment by UserWangZz** [comment, marked as answer]
The annotation file format supported by the text detection algorithm in PaddleOCR is as follows, separated by '\t':

```
" image path                    Image annotation information encoded by json.dumps"
ch4_test_images/img_61.jpg    [{"transcription": "MASA", "points": [[310, 104], [416, 141], [418, 216], [312, 179]]}, {...}]
```
We do not provide tools for data conversion, but you can easily convert the yolo format to the paddleocr detection format.

If you do not have a field for text content, you can set the transcription field to an empty character, such as "".
Text detection reference document: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/detection_en.md
Text recognition reference document: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/recognition_en.md

**Reply by wdcs-krishpatel** [reply]
Okkay now I got it. Thank you @UserWangZz  for the support.
